src/Pact/Types/Parser.hs

```diff
@@ -47,3 +48,13 @@ style = IdentifierStyle "atom"
         (HS.fromList ["true","false"])
         Symbol
         ReservedIdentifier
+
+-- helper function used when generating arbitrary strings
```

Comment:
Move this to usage site in `Util` to leave this module unchanged, where you can construct it without a top-level def from `style`, ie

```
where 
   style' = style { _styleStart = (letter <|> digit <|> symbols) }
```

Reply:
See latest commit